packages/gatsby-remark-images/src/index.js

```diff
@@ -172,7 +172,7 @@ module.exports = (
   <span
     class="gatsby-resp-image-wrapper"
     style="position: relative; display: block; ${
-      options.wrapperStyle
+      options.showCaptions && node.title ? null : options.wrapperStyle 
```

Comment:
Nitpicky - can we do `const showCaption = options.showCaptions && node.title` before this and then use that both here and in L204 to not multiplicate same logic to multiple places?

Other than that, it looks great!